docs/source/examples/response/timelines/post/simple_with_monthly.json

```diff
@@ -170,22 +198,22 @@
                 "mbtu": [0.0]
               },
               "total": {
-                "mbtu": [0.0]
+                "mbtu": [204.29]
               }
             },
             "monthly": {
               "coalTotal": {
                 "mbtu": [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
               },
               "electricityTotal": {
-                "kwh": [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-                "mbtu": [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
+                "mbtu": [2.93, 2.44, 2.15, 1.75, 2.21, 2.16, 3.52, 2.91, 1.94, 1.68, 1.98, 2.70],
+                "kwh": [859.41, 714.81, 629.55, 511.45, 648.17, 632.72, 1032.89, 853.19, 568.66, 493.00, 579.91, 790.64]
```

Comment:
@kellyatradiantlabs Now I'm confused. Does `MBtu` stands for million Btu's?

It made sense to me when I changed the source units like in this screenshot.

![image](https://github.com/RadiantLabs/modeling-api-docs/assets/10285345/6a911518-33d3-490f-842a-20fe5e3be725)

Reply:
lol imperial units are so counterintuitive and the labels between kBtu, mBtu/Mbtu and MMBtu are really inconsistent. In our platform, any reference to `mbtu` means million btu. 

For further explanation, sometimes people in US use MMBtu for million btu and MBtu for thousand btu (which makes no sense in my opinion). I'm trying to avoid thousand Btu in our API if at all possible, although some of the hourly results returned from OpenStudio come in kBtu (also thousand Btu) currently.

Reply:
I see! Wow, that's wild. Then when we see `kBtu` coming from OS, we'll need to use the decimal representation of MMBtu to keep units consistent? Right now, we only support `mbtu` in the API, which means millions of btu according to your explanation.

Another question came up. Do hourly results come in kBtu always or sometimes? I looked at 501 Randall's hourly data and all references to btu are in kBtu.

Reply:
When we receive `kbtu` (thousand Btu) from OS, I think we should just convert to `mbtu` (million btu) before returning results to users to be consistent. 

I believe all hourly results come from OS in thousand btu and all annual results from OS are in million btu.